mlrun/api/main.py

```diff
@@ -431,7 +431,10 @@ async def _verify_log_collection_stopped_on_startup():
             db_session,
             requested_logs_modes=[True],
             only_uids=False,
-            states=mlrun.runtimes.constants.RunStates.terminal_states(),
+            states=mlrun.runtimes.constants.RunStates.terminal_states()
+            + [
+                mlrun.runtimes.constants.RunStates.unknown,
+            ],
```

Comment:
this is the kind of the scenarios where we wish a comment existed for why we suddenly care for specific case. please add :)